src/App.tsx

```diff
@@ -1,12 +1,20 @@
 // import { useState } from 'react';
+import { Routes, Route } from 'react-router-dom';
 import './App.css';
 import Login from './pages/login/Login';
-// import Registration from './pages/registration/Registration';
+import HomePage from './pages/home/home';
+import Registration from './pages/registration/Registration';
+import Page404 from '../src/pages/page404/page404';
 
 function App() {
   return (
     <>
-      <Login />
+      <Routes>
+        <Route path="/" element={<HomePage />} />
+        <Route path="/login" element={<Login />} />
+        <Route path="/register" element={<Registration />} />
+        <Route path="/error404" element={<Page404 />} />
+      </Routes>
```

Comment:
It is possible to add processing of non-existent routes.

```suggestion
      <Routes>
        <Route path="/" element={<HomePage />} />
        <Route path="/login" element={<Login />} />
        <Route path="/register" element={<Registration />} />
        <Route path="/error404" element={<Page404 />} />
        <Route path="/*" element={<Page404 />} />
      </Routes>
```